src/components/Shop/Shop.js

```diff
@@ -123,7 +123,7 @@ export const Shop = ({
                 />
                 <CardContent>
                   <Typography>
-                    Purchase a Storge Unit to increase your inventory capacity
+                    Purchase a Storage Unit to increase your inventory capacity
```

Comment:
🤦‍♂️

Thank you for spotting and fixing this! I definitely didn't mean this, per Wikipedia

>Storge (/ˈstɔːrɡi/, from the Ancient Greek word στοργή storgē) or familial love refers to natural or instinctual affection, such as the love of a parent towards offspring and vice versa.